py-polars/polars/convert.py

```diff
@@ -34,7 +34,7 @@ def from_dict(
     columns: Sequence[str] | None = None,
 ) -> DataFrame:
     """
-    Construct a DataFrame from a dictionary of sequences.
+    Construct a DataFrame from a dictionary of sequences. This operation clones data.
```

Comment:
In this case it may clone data. You can also pass a `dict[str, pl.Series]`. This does not clone data.

Reply:
I see. I guess this also applies to `pl.from_dicts`?